packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -44,12 +44,15 @@ class FlutterDevice {
     _viewsCache = null;
   }
 
-  void connect() {
+  /// If the [reloadSources] parameter is not null the 'reloadSources' service
+  /// will be registered
```

Comment:
Could you elaborate on this documentation? Suppose someone doesn't know what reloadSources are. The current documentation tells them very little.